benchmark/benchmain/main.go

```diff
@@ -137,13 +138,31 @@ func makeFuncUnary(benchFeatures stats.Features) (func(int), func()) {
 		)
 	}
 	sopts = append(sopts, grpc.MaxConcurrentStreams(uint32(benchFeatures.MaxConcurrentCalls+1)))
-	opts = append(opts, grpc.WithDialer(func(address string, timeout time.Duration) (net.Conn, error) {
-		return nw.TimeoutDialer(net.DialTimeout)("tcp", address, timeout)
-	}))
 	opts = append(opts, grpc.WithInsecure())
 
-	target, stopper := bm.StartServer(bm.ServerInfo{Addr: "localhost:0", Type: "protobuf", Network: nw}, sopts...)
-	conn := bm.NewClientConn(target, opts...)
+	var lis net.Listener
+	if *useBufconn {
+		bcLis := bufconn.Listen(256 * 1024)
```

Comment:
Make this a little bit bigger just to be safe. Our I/O buffers for benchmark tests are 128K each side but the new changes may result in writing data a little over that limit.

Reply:
As discussed offline, this buffer is not shared between the two directions, there are two separate buffers of 256K each.